index.html

```diff
@@ -0,0 +1,968 @@
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                Why our laws can't protect me from my digital stalker
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                Why our laws can't protect me from my digital stalker
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                Why our laws can't protect me from my digital stalker
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                Why our laws can't protect me from my digital stalker
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div>
+          <div class="title">
+            <h1>Latest news</h1>
+            <i class="fas fa-angle-right"></i>
+          </div>
+          <div class="grid-background sectionflex">
+            <h3>MOST POPULAR</h3>
+            <p>
+              Microsoft once owned a chunk of Apple, here’s what it would have
+              been worth today
+            </p>
+            <p class="second-p"><a href="#">Rachel Kaser</a> - 13hours ago</p>
+          </div>
+          <div class="grid-flex sectionflex">
+            <div>
+              <h4 class="red">LATEST</h4>
+              <p>
+                <a href="#" class="red"
+                  >Satoshi Nakaboto: ‘Ryanair CEO believes Bitcoin is a ponzi
+                  scheme’</a
+                >
+              </p>
+            </div>
+            <div class="flex-img">
+              <img src="./images/chinaman.jpg" alt="img" />
+            </div>
+          </div>
+          <ul>
+            <li>
+              <a href="#">
+                Why our laws can't protect me from my digital stalker
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                Why our laws can't protect me from my digital stalker
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                Why our laws can't protect me from my digital stalker
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                Why our laws can't protect me from my digital stalker
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                Why our laws can't protect me from my digital stalker
+              </a>
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <!-- Section 5 -->
+
+      <section class="section5 alignment">
+        <div class="title flex">
+          <div>
+            <h1>Latest Deals</h1>
+            <i class="fas fa-angle-right"></i>
+          </div>
+          <div>
+            <a class="red" href="#">See all deals</a>
+          </div>
+        </div>
+        <div class="grid grid-divs">
+          <div>
+            <img src="./images/suit.jpg" alt="image" />
```

Comment:
#### Mobile Screen Size

- [ ] Here the layout is different from the original. Kindly address that.

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/91437213-e9b8af80-e83f-11ea-855c-0f1962054e32.png) | ![image](https://user-images.githubusercontent.com/61479648/91437235-f3421780-e83f-11ea-9ec5-04a61a631049.png) |